src/Castle.ts

```diff
@@ -0,0 +1,198 @@
+import fetch from 'node-fetch';
+import { reduce } from 'lodash';
+import { IncomingHttpHeaders } from 'http2';
+
+const apiUrl = 'https://api.castle.com';
```

Comment:
could this be configurable? For staging, local execution, etc

Reply:
Ah sure, we should probably update the SDK specification to include that too

Reply:
The url is wrong, it should be `api.castle.io`

Reply:
My bad, it used to be right :(

Reply:
Did you use this library after changes? In an example app or on its own?

Reply:
I didn't use it after the changes I did today, no

Reply:
@nijikon yeah works.

The little app I have for testing this stuff is pretty bad though, I'm writing a new one. Would be nice to have an app in `examples` to show it off.

Reply:
OK, it changed in the middle of commits to `api.castle.com`. Can you test it with all the recent changes or did you already?

Reply:
I did :)

Reply:
Thanks!